_tests/integration/docker_pull_bitrise.yml

```diff
@@ -0,0 +1,71 @@
+format_version: 1.3.0
+default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
+workflows:
+  docker-pull-success:
+    title: Expected to pass docker pull
+    container:
+      image: frolvlad/alpine-bash:latest
+    steps:
+    - script:
+        title: Should pass
+        inputs:
+        - content: exit 0
+  docker-pull-fails-404:
+    title: Expected to fail docker pull
+    container:
+      image: nowhere/noimage:latest
```

Comment:
Nightmare: someone publishes this image to DockerHub and we'll have to deal with a failing test 😆 At least please us an impossible tag, such as a checksum